tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -508,6 +508,24 @@ public function userSendsTheFollowingSpaceShareInvitationToFederatedUserUsingPer
 		);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has updated the last resource share with:$/
```

Comment:
```suggestion
	 * @Given user :user has updated the last resource share with:
```

Reply:
@S-Panta why you think this matter plz elaborate